app/components/events/event_box_component.rb

```diff
@@ -7,6 +7,14 @@ class EventBoxComponent < ViewComponent::Base
     alias_method :virtual?, :virtual
 
     def initialize(event)
+      # An event can have three different 'states':
+      # - Offline (in person)
+      #   - is_online == false && is_virtual == false
+      # - Moved online (online, but has an associated building - aka virtual)
+      #   - is_online == false && is_virtual == true
```

Comment:
If an event has moved online it should be is_online == true && is_virtual == true

Reply:
Ah - good call